Doc/library/decimal.rst

```diff
@@ -571,29 +585,10 @@ Decimal objects
       >>> Decimal(321).exp()
       Decimal('2.561702493119680037517373933E+139')
 
-   .. method:: from_float(f)
+   .. classmethod:: from_float(f)
 
-      Classmethod that converts a float to a decimal number, exactly.
-
-      Note `Decimal.from_float(0.1)` is not the same as `Decimal('0.1')`.
-      Since 0.1 is not exactly representable in binary floating point, the
-      value is stored as the nearest representable value which is
-      `0x1.999999999999ap-4`.  That equivalent value in decimal is
-      `0.1000000000000000055511151231257827021181583404541015625`.
-
-      .. note:: From Python 3.2 onwards, a :class:`Decimal` instance
```

Comment:
This note is very dated and can be substantially reduced but not eliminated.  Otherwise, it is very confusing the this method exists at all, so it is helpful to a reader to know that its use cases have been almost entirely supplanted.